### PR TITLE
ISV partners revamp

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated ISV partners page as per [doc](https://docs.google.com/document/d/1qWWpFbwZykjAIt-Kf6iFCXPCnmmuKBbU5mzvILnO-q0/edit) and [mock-up](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=12110%3A16698&mode=dev)
- Added mention to ISV page on partners landing as per [doc](https://docs.google.com/document/d/1SwCceQv625Kdc3wAvzSFBBA3qt3CxzDmc0DAfo3LJfs/edit)
- Updated partners subnav and shared footer

## QA

- View the site locally in your web browser at: https://canonical-com-1154.demos.haus/partners
- See that the link to the [ISV partners page](https://canonical-com-1154.demos.haus/partners/isv-partners) exists, matches relevant doc, and routes correctly
- Check ISV partners page against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5266

### CODE REVIEW

**Comment by juanruitina** [comment]
UX+1!

**Comment by lyubomir-popov** [comment]
looks good, just one thing - can we please have a 50/50 split - h5 titles on the left, ticked lists on the right - on medium screens, to match the hero: 
image

File: templates/partners/isv-partners.html [hunk 1]
```diff
@@ -6,44 +6,75 @@
...
+{% endwith %}
+
+<section class="p-section">
+  <div class="row--50-50">
+    <div class="col">
+      <h1>ISV Embedding<br class="u-hide--small"> Programme</h1>
+    </div>
+    <div class="col">
+      <div class="p-block">
+        <p>From Ubuntu Server to containers, Canonica's portfolio is used by independent software vendors (ISVs) around the world to power SaaS, appliances and software solutions deployed within public clouds, data -centres, edge infrastructure and workstations. Our ISV Embedding programme offers the optimal technical, commercial and legal framework to enjoy cost-effective support across the lifecycle of your product.</p>
```
Comment (carkod) [inline templates/partners/isv-partners.html:24]: Missing "l" in "Canonica's"

File: templates/partners/partial/_footer.html [hunk 2]
```diff
@@ -1,30 +1,11 @@
-<section class="p-section l-footer--sticky">
-  <div class="row">
+<section class="p-strip--white is-deep l-footer--sticky">
```
Comment (carkod) [inline templates/partners/partial/_footer.html:1]: Do we need `l-footer--sticky`? Nothing happens if I remove it.

Comment (mtruj013) [reply templates/partners/partial/_footer.html]: Nope, removed!

**Pull request merged**